source/PlayerInfo.cpp

```diff
@@ -1136,17 +1137,19 @@ void PlayerInfo::Land(UI *ui)
 	// Bring auto conditions up-to-date for missions to check your current status.
 	UpdateAutoConditions();
 	
-	// Update missions that are completed, or should be failed.
+	// Update missions that are completed, should be failed, or should be made inactive.
 	StepMissions(ui);
 	UpdateCargoCapacities();
 	
-	// Create whatever missions this planet has to offer.
+	// If the player is actually landing (rather than simply loading the game),
+	// new missions are created and new fines may be levied.
 	if(!freshlyLoaded)
+	{
+		// Create whatever missions this planet has to offer.
 		CreateMissions();
-	
-	// Check if the player is doing anything illegal.
-	if(!freshlyLoaded)
+		// Check if the player is doing anything illegal.
 		Fine(ui);
```

Comment:
Nice fix. Also unrelated to the change in this PR. It is easier for reviewers if you put fixes like this in separate PRs.